### ISSUE TITLE
Acrosomal hypoplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9885]
**Preferred term label:**
Acrosomal hypoplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduction in the size of the acrosome of the head of spermatozoa.

PMID:3311822

The acrosome is a unique membranous organelle located over the anterior part of the sperm nucleus that is highly conserved throughout evolution. This acidic vacuole contains a number of hydrolytic enzymes that, when secreted, help the sperm penetrate the egg's coats. See Figure 23 of PMID:3311822

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271650/
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620277 
PMID: 3670803  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MagoWyr  -- I found this term in a disease entry I am annotating and drafted the above definition. Is this accurate?

**Comment by MagoWyr** [comment]
Yes, the definition is correct!
________________________________

**Comment by pnrobinson** [comment]
thanks!
added
